tests/sentry/grouping/test_normalize_message.py

```diff
@@ -119,25 +119,30 @@ def record_analytics(monkeypatch):
             """blah connection failed after 12345ms 1.899s 3s""",
             """blah connection failed after <duration> <duration> <duration>""",
         ),
+        # (
+        #     "Uniq ID - sql savepoint",
+        #     '''SQL: RELEASE SAVEPOINT "s140177518376768_x2"''',
+        #     '''SQL: RELEASE SAVEPOINT "<uniq_id>"''',
+        # ),
+        # (
+        #     "Uniq ID - api gateway",
+        #     """API gateway VdLchF7iDo8sVkg= blah""",
+        #     """API gateway <uniq_id>= blah""",
+        # ),
+        # (
+        #     "Uniq ID - fb trace",  # TODO: It is possible to have fbtrace_ids without integers.
+        #     """fbtrace_id Aba64NMEPMmBwi_cPLaGeeK AugPfq0jxGbto4u3kxn8u6p blah""",
+        #     """fbtrace_id <uniq_id> <uniq_id> blah""",
+        # ),
+        # (
+        #     "Uniq ID - word with numerical pre/suffix",
+        #     """1password python3 abc123 123abc""",
+        #     """1password python3 abc123 123abc""",
+        # ),
         (
-            "Uniq ID - sql savepoint",
-            '''SQL: RELEASE SAVEPOINT "s140177518376768_x2"''',
-            '''SQL: RELEASE SAVEPOINT "<uniq_id>"''',
-        ),
-        (
-            "Uniq ID - api gateway",
-            """API gateway VdLchF7iDo8sVkg= blah""",
-            """API gateway <uniq_id>= blah""",
-        ),
-        (
-            "Uniq ID - fb trace",  # TODO: It is possible to have fbtrace_ids without integers.
-            """fbtrace_id Aba64NMEPMmBwi_cPLaGeeK AugPfq0jxGbto4u3kxn8u6p blah""",
-            """fbtrace_id <uniq_id> <uniq_id> blah""",
-        ),
-        (
-            "Uniq ID - word with numerical pre/suffix",  # TODO: It is possible to have fbtrace_ids without integers.
-            """1password python3 abc123 123abc""",
-            """1password python3 abc123 123abc""",
+            "UUID after underscore",
+            "[words] look come-look_18d34d42-1aaa-6bac-bce3-4c4a854061d2: true",
+            "[words] look come-look<uuid>: true",  # TODO: Should we parameterize bools like this?
```

Comment:
Is this reverting to the original values? It's hard to understand from the diff if we're just reverting or not.